main.rb

```diff
@@ -0,0 +1,53 @@
+require './app'
+
+class Main
+  def self.menu
+    puts 'Welcome to the Library'
+    puts 'Please enter a number to select the option of your choice from the following list'
+
+    @list = {
+      '1' => 'List of All Books',
+      '2' => 'List of All Peoples',
+      '3' => 'Create a Person',
+      '4' => 'Create a Book',
+      '5' => 'Create a Rental',
+      '6' => 'List of All Rentals for a person ID',
```

Comment:
Great job @chuaindia 🤝

- [ ] You may kindly fix the listing of rentals. option 6. When I try to list rentals, I get the error below.
![error](https://user-images.githubusercontent.com/63917725/210637392-417512bf-511a-404b-a9f0-f3cbc701c894.PNG)

Reply:
Done accordingly.